app/javascript/components/mentoring/mentor_conversations_list/pagination.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+
+export default function Pagination({ current, total, setPage, around = 3 }) {
+  const range = new Range(
+    Math.max(current - around, 1),
+    Math.min(current + around, total)
+  )
+
+  function Range(start, end) {
+    return Array(end - start + 1)
+      .fill()
+      .map((_, i) => start + i)
+  }
+
+  return (
+    <div>
+      <button
+        onClick={() => {
+          setPage(1)
+        }}
+        disabled={current === 1}
```

Comment:
We should add an `aria-label` here. We will read this as `First`, which isn't too bad, but users using a screen users won't have the same _visual_ context.

```
aria-label="Go to first page"
```

Add `aria-current="page"` if the page is current.